packages/lib/src/components/FeedbackWidget/FeedbackWidget.tsx

```diff
@@ -229,6 +249,7 @@ export function FeedbackWidget({
         w="390px"
         h={openInput ? 'auto' : '180px'}
         position={{ bottom: 100, right: 80 }}
+        className={isCapturingScreenshot ? classes.displayNone : ''}
```

Comment:
Here I'd suggest to use another class name since we want to follow BEM.

Reply:
Good point 🎾